lib/components/base-components/Renderable.ts

```diff
@@ -197,18 +213,23 @@ export abstract class Renderable implements IRenderable {
       if (hasIncompleteEffects) return
     }
 
+    // Handle start
+    emitRenderEvent("start")
```

Comment:
Similar to my other comment, i don't like how the phase is implicit here, I'd prefer if the phase was explicitly specified